pkgs/by-name/ka/kanidm/package.nix

```diff
@@ -78,6 +78,9 @@ rustPlatform.buildRustPackage rec {
     rust-jemalloc-sys
   ];
 
+  # can take over 4 hours on 2 cores and needs 16GB+ RAM
+  requiredSystemFeatures = [ "big-parallel" ];
+
```

Comment:
nitpick: In most packages, this is placed before `passthru` or `meta`, currently it’s a bit in the middle between build-recipe related things.

Reply:
Cool, moved. It seemed build related to me, but I'm not picky on the location.